docs/wiki/protocol/pm.md

```diff
@@ -0,0 +1,13 @@
+## Ethereum Project Management Repository
```

Comment:
This page should not be just about the pm repo but coordination of core devs in general. Sorry I wasn't clear about it. PM repo is still part of that so this is ok, just needs an intro and another h1